### PR TITLE
[daint-gpu] Addition of GROMACS-PLUMED recipe

### PR DESCRIPTION
**Pull request by omlins**

### CODE REVIEW

**Comment by omlins** [comment]
retest this please

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2016.3-CrayGNU-17.08-PLUMED-2.3.0-cuda-8.0.61.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-93.html
+#
+name = 'GROMACS'
+version = "2016.3"
+cudaversion = '8.0'
+versionsuffix = '-PLUMED-2.3.0-cuda-%s' % cudaversion
```
Comment (gppezzi) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2016.3-CrayGNU-17.08-PLUMED-2.3.0-cuda-8.0.61.eb]: cudaversion is `8.0`, so could you please rename the file in order to match that? (remove the `.61`)

File: jenkins-builds/6.0.UP04-17.08-gpu [hunk 2]
```diff
@@ -15,6 +15,7 @@
  Extrae-3.5.2-CrayIntel-17.08.eb
  GREASY-2.1-cscs-CrayGNU-17.08.eb                   --set-default-module
  GROMACS-2016.3-CrayGNU-17.08-cuda-8.0.eb           --set-default-module
+ GROMACS-2016.3-CrayGNU-17.08-PLUMED-2.3.0-cuda-8.0.61.eb  --set-default-module
```
Comment (gppezzi) [inline jenkins-builds/6.0.UP04-17.08-gpu]: Is this a request from a single user? 

If this is for a single user, we should only provide him the easyconfig file and he should install it by himself.

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2016.3-CrayGNU-17.08-PLUMED-2.3.0-cuda-8.0.61.eb (hunk 1 above)
Comment (omlins) [reply easybuild/easyconfigs/g/GROMACS/GROMACS-2016.3-CrayGNU-17.08-PLUMED-2.3.0-cuda-8.0.61.eb]: Done.

File: jenkins-builds/6.0.UP04-17.08-gpu (hunk 2 above)
Comment (omlins) [reply jenkins-builds/6.0.UP04-17.08-gpu]: I removed the line again.

**Pull request merged**